### PR TITLE
Add MS SQL Server DB Client

### PR DESCRIPTION
**Pull request by Neaj-Morshad-101**

### CODE REVIEW

File: mssql/client.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+limitations under the License.
+*/
+
+package mssql
+
+import (
+	"xorm.io/xorm"
+)
+
+type XormClient struct {
```
Comment (ArnobKumarSaha) [inline mssql/client.go:23]: Add a utility func `Close()` to close the xorm client.

File: go.mod [hunk 2]
```diff
@@ -25,11 +25,13 @@ require (
 	k8s.io/klog/v2 v2.120.1
 	kmodules.xyz/client-go v0.29.13
 	kmodules.xyz/custom-resources v0.29.1
-	kubedb.dev/apimachinery v0.44.1-0.20240418111017-ee6ba81797f6
+	kubedb.dev/apimachinery v0.44.1-0.20240425042236-6efef42b8792
 	sigs.k8s.io/controller-runtime v0.17.2
 	xorm.io/xorm v1.3.6
 )
 
+require github.com/microsoft/go-mssqldb v1.6.0
```
Comment (ArnobKumarSaha) [inline go.mod:33]: Move this to above require block

File: mssql/kubedb_client_builder.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+	//			RootCAs:      certPool,
+	//			Certificates: clientCert,
+	//		})
+	//		tlsConfig = fmt.Sprintf("tls=%s", api.MSSQLTLSConfigCustom)
+	//	} else {
+	//		tlsConfig = fmt.Sprintf("tls=%s", api.MSSQLTLSConfigSkipVerify)
+	//	}
+	//}
+
+	connectionString := fmt.Sprintf("server=%s;user id=%s;password=%s;database=%s;%s", o.url, user, pass, "master", tlsConfig)
```
Comment (ArnobKumarSaha) [inline mssql/kubedb_client_builder.go:156]: If database is always `master`, we can use it like `database=master` directly

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: mssql/client.go (hunk 1 above)
Comment (Neaj-Morshad-101) [reply mssql/client.go:23]: added

**Review by ArnobKumarSaha — APPROVED** [review]

**Pull request merged**